### ISSUE TITLE
copy full history (feature suggestion)

### ISSUE THREAD

**Issue by etamtlosz** [thomasnordquist/MQTT-Explorer#388]
I'd like to have an option to copy/export the whole history. Right now is possible to copy every item one by one.
Thanks

**Comment by AgentBignose** [comment]
Yes json for array for boxed and csv for raw that would be great!

**Comment by Andoramb** [comment]
I agree, so far the only thing missing is the export capability 👍
